client/layout/style.scss

```diff
@@ -72,135 +115,140 @@
 	}
 }
 
-.layout__primary {
-	transition: all 0.15s ease-in-out;
+// Setup the secondary element, which contains the sidebar and
+// the site-selector elements.
+.layout__secondary {
+	position: fixed;
+		top: 47px;
+		left: 0;
+		bottom: 0;
+	color: var( --sidebar-color );
+	background: var( --sidebar-background );
+	border-right: 1px solid darken( $sidebar-bg-color, 5% );
+	width: $sidebar-width-max;
+	overflow: hidden;
 
-	.is-section-preview & {
-		height: 100%;
+	@include breakpoint( "<960px" ) {
+		width: $sidebar-width-min;
 	}
-}
 
-.layout__content a {
-	text-decoration: none;
-}
+	@include breakpoint( "<660px" ) {
+		width: 100%;
+	}
 
-.layout {
-	.sidebar,
-	.layout__secondary .site-selector,
-	.current-site,
-	.sidebar__menu {
-		transform: translateX( 0 );
-		transition: all 0.15s cubic-bezier(0.075, 0.820, 0.165, 1.000);
+	// The editor has its own sidebar which doesnt
+	// use the secondary element.
+	.is-section-post-editor & {
+		display: none;
 	}
 }
 
-.layout {
-	.sidebar,
-	.masterbar {
-		@media print {
-			display: none;
+// Setup the site-selector element. Its default
+// position is off screen.
+.layout__secondary .site-selector {
+	position: absolute;
+		top: 0;
+		right: 0;
+		bottom: 0;
+		left: 0;
+	pointer-events: none;
+	transform: translateX( -$sidebar-width-max );
+	pointer-events: none;
```

Comment:
Duplicate `pointer-events` rule. (Note that i'm not a huge fan of modifying this value)

Reply:
+1 for not modifying `pointer-events`.